pkg/scannerv4/mappers/mappers.go

```diff
@@ -699,52 +696,6 @@ func severityAndScores(ctx context.Context, vuln *claircore.Vulnerability, nvdVu
 	return nvdSeverityAndScores(vuln, nvdVuln)
 }
 
-func rhelSeverityAndScores(vuln *claircore.Vulnerability) (severityValues, error) {
```

Comment:
Just realized that, if you want to leave the door open to rollback CSAF/VEX if things doesn't go as expected, then you might want to keep this function around as well.

Reply:
https://github.com/stackrox/stackrox/pull/12452/commits/b94446c77b0bff7f966050d8cbee876094ca1ba2